### ISSUE TITLE
A small bug

### ISSUE THREAD

**Issue by DNALuo** [qfgaohao/pytorch-ssd#67]
Great implementation! 
But I found there is a bug, it is about the result "locations" not at the same device as "priors" .
Just add one line after line 91 in "pytorch-ssd/vision/ssd/ssd.py" and everything works perfect. 
```python
if self.is_test:
            confidences = F.softmax(confidences, dim=2)
            locations = locations.to(self.device)
            boxes = box_utils.convert_locations_to_boxes(
                 locations, self.priors, self.config.center_variance, self.config.size_variance
            )

**Comment by qfgaohao** [comment]
Thanks, @DNALuo . You are right. They should be on the same device.
